canonical/canonical.com
# WD-3107 build new executive summit page

## Done

- Build the new executive summit page

## QA

- Check out the [demo here](https://canonical-com-865.demos.haus/partners/executive-summit) and compare it with the [copy doc](https://docs.google.com/document/d/15C3k-htvapwp-WTrYQ2SdHBW20Lp9IinpY9mkvo6q3U/edit) and the [design](https://www.figma.com/file/6wmLn4OgDDAgfpmkLFqfp5/c.com%2Fpartners%2Fexecutive-summit?node-id=0%3A1&t=0PNCjIEAdjiJfmKb-1).

## Issue / Card

- [WD-3107](https://warthogs.atlassian.net/browse/WD-3107)


[WD-3107]: https://warthogs.atlassian.net/browse/WD-3107?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
